plugins/ReverbSC/dcblock.c

```diff
@@ -26,11 +27,11 @@ int sp_dcblock_destroy(sp_dcblock **p)
     return SP_OK;
 }
 
-int sp_dcblock_init(sp_data *sp, sp_dcblock *p)
+int sp_dcblock_init(sp_data *sp, sp_dcblock *p, int oversampling )
 {
     p->outputs = 0.0;
     p->inputs = 0.0;
-    p->gain = 0.99;
+    p->gain = pow( 0.99, 1.0f / oversampling );
```

Comment:
The idea here is that the oversampling causes the net gain to vary which changes the frequency response of the DC block. Now we always get a gain of 0.99 .